server/src/api/customers.ts

```diff
@@ -58,6 +58,12 @@ customerRouter.get(
 customerRouter.post(
```

Comment:
I think you need to update the function comment to include more things that this function is doing.

Reply:
@xlai20 Oh right! Ok sure!